src/src/main/java/dao/impl/UsersDAO.java

```diff
@@ -0,0 +1,34 @@
+package dao.impl;
+
+import dao.UsersDAOInterface;
+import service.DataBaseService;
+import model.User;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
+public class UsersDAO implements UsersDAOInterface {
+
+    public UsersDAO() {
+    }
+
+    public static User create(String email, String login, String password) {
+        if (Objects.isNull(email)
+                || Objects.isNull(login)
+                || Objects.isNull(password)) {
+            throw new NoSuchElementException("Wrong parameters, try again!");
+        }
+        return new User(email, login, password);
+    }
+
+    @Override
+    public void add(Object o) {
```

Comment:
```suggestion
    public void add(User o) {
```

Reply:
fixed all other problems; we can`t make add (User o), because this method is an interface implementation.

Reply:
why we can't do it?